H/HiGHS/highs_common.jl

```diff
@@ -18,9 +18,13 @@ sources = [
 platforms = supported_platforms()
 
 function build_script(; shared_libs::String)
-    return "BUILD_SHARED=$(shared_libs)\n" * raw"""
+    build_static = shared_libs == "OFF" ? "ON" : "OFF"
+    return "BUILD_SHARED=$(shared_libs)\nBUILD_STATIC=$(build_static)\n" * raw"""
 cd $WORKSPACE/srcdir/HiGHS
 
+# Remove system CMake to use the jll version
+apk del cmake
```

Comment:
I'll need to remember to add `HostBuildDependency(PackageSpec(; name="CMake_jll")),` to `HiGHS_jll.jl the next time I rebuild it, but that's okay. I'll likely find this comment when I go searching if I forget.